src/__test__/homepage.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+import App from '../components/App';
+import MapComponent from '../components/site/LandingPage/MapComponent';
+
+it(">>>> Whole App component should render correctly", () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+it(">>>> Map renders correctly", () => {
+  const { getByTestId } = render(<MapComponent lat={-1.9470658} lng={30.0915372}></MapComponent>);
+  expect(getByTestId('map-component')).toBeValid();
+});
```

Comment:
These types of errors shouldn't be found in your code any more

Reply:
Good job. But you have to increase on your test coverage

Reply:
@Malaba6 yes let me work on that now